packages/core/strapi/src/services/entity-validator/blocks-validator.ts

```diff
@@ -27,14 +27,10 @@ const linkNodeValidator = yup.object().shape({
 // TODO: remove any with a correct Type
 const inlineNodeValidator: any = yup.lazy((value: { type: string }) => {
   switch (value.type) {
-    case 'text':
-      return textNodeValidator;
     case 'link':
       return linkNodeValidator;
     default:
-      return yup.mixed().test('invalid-type', 'Inline node must be Text or Link', () => {
-        return false;
-      });
```

Comment:
I'm not really opposed but I'm wondering what your reasoning is for changing this?